dataactvalidator/config/sqlrules/a30_appropriations.sql

```diff
@@ -0,0 +1,16 @@
+SELECT approp.row_number,
+	approp.allocation_transfer_agency,
+	approp.agency_identifier,
+	approp.beginning_period_of_availa,
+	approp.ending_period_of_availabil,
+	approp.availability_type_code,
+	approp.main_account_code,
+	approp.sub_account_code
+FROM appropriation AS approp
+WHERE approp.submission_id = {}
+	AND NOT EXISTS (
+		SELECT 1
+		FROM object_class_program_activity AS op
+		WHERE approp.tas IS NOT DISTINCT FROM op.tas
```

Comment:
Yes, it definitely should include submission ID, we don't want to have matches to past submissions satisfy this rule.